server/index.js

```diff
@@ -30,3 +37,25 @@ app.put('/api/nearby/:carousel_id', (req, res) => {
     .then(updated => res.status(202).send(updated.carousel))
     .catch(err => res.status(400).json(err));
 });
+
+app.put('/api/nearby/:carousel_id', (req, res) => {
+  db.addFavorite(req.params.carousel_id, req.query.restaurantId, req.query.increment)
+    .then(updated => res.status(202).send(updated.carousel))
+    .catch(err => res.status(400).json(err));
+});
+
+app.put('/api/nearbyUpdate/', (req, res) => {
+  const carouselUpdate = req.body;
+  const { id, carousel } = carouselUpdate;
```

Comment:
Here I would cut out line 48 and just say `const { id, carousel } = req.body`.

Reply:
Good call thanks!